generators/Exercises/ExerciseCollection.cs

```diff
@@ -48,6 +52,20 @@ private IEnumerable<Exercise> GetExercises()
 
         private bool IsNotImplemented(string exerciseName) => !_exerciseTypesByName.ContainsKey(exerciseName.ToExerciseName());
 
+        private bool IsOutdated(string exerciseName) {
+            var filePath = FilePathHelper.TestClassFilePath(exerciseName, exerciseName.ToTestClassName());
+            var firsLine = File.ReadLines(filePath).First();
+
+            if(firsLine.StartsWith("//")) {
```

Comment:
Teeny tiny nit: there should be a space after the `if`.

Reply:
Fixed! :)